src/modules/endpoint/sagas/fetchResourceCache.js

```diff
@@ -53,27 +49,31 @@ export default function * fetchResourceCache (city: string, language: string, fe
     const failureResults = pickBy(results, result => !!result.errorMessage)
     if (!isEmpty(failureResults)) {
       const message = createErrorMessage(failureResults)
-
       const failed: ResourcesFetchFailedActionType = {type: `RESOURCES_FETCH_FAILED`, city, language, message}
+      console.warn(message)
+      // todo: we might remember which files have failed to retry later (internet connection of client could have failed)
```

Comment:
Feel free to create a task for that todo.

Reply:
done